addons/website_slides/static/src/scss/website_slides_profile.scss

```diff
@@ -2,8 +2,4 @@
 .o_wprofile_slides_course_card_body {
     cursor: pointer;
 
-    .o_wslides_background_image {
-        background-size: cover;
-        background-position: center;
-    }
```

Comment:
Is this removing a duplication? or the actual rules (because we do still use `o_wslides_background_image`)